es6.md

```diff
@@ -145,3 +155,37 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+# Modules
+
+## ExportDeclaration
+
+```js
+interface ExportDeclaration <: Declaration {
+    default: boolean;
+    declaration: VariableStatement | Declaration | HoistableDeclaration | null;
```

Comment:
Also, as @RReverser notes, we need to support the ES6 definition of an `assignmentExpression` which includes a `conditionalExpression` which includes all binary expressions, like `export default (1 + 42);`